app/build.gradle.kts

```diff
@@ -33,12 +29,21 @@ android {
             )
         }
     }
+
+    buildFeatures {
+        viewBinding = true
+        compose = true
+    }
+
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_11
-        targetCompatibility = JavaVersion.VERSION_11
+        sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
How many users are using a version between 11 and 16?
(meaning: how many curent and future users will we lose?)